cinema_service/settings.py

```diff
@@ -116,6 +116,12 @@
 
 AUTH_USER_MODEL = "user.User"
 
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.TokenAuthentication",
+    ]
+}
```

Comment:
you can set default permissions here as well and override it in views where it should be different